### ISSUE TITLE
Use of Template Rendering library to seprate the Python and HTML Code

### ISSUE THREAD

**Issue by RamolaWeb** [CuriousLearner/GeeksForGeeksScrapper#16]
The Current Script to scrape the website contains both python as well as the HTML code . Thus in order to scale the project i.e to modify the HTML code to redesign the Web Page and PDF document, we may need to modify the Script also. Thus it will great to use a template rendering library like **Jinja2** to render the HTML content so that the HTML and Python Part Can be separated.

**Comment by CuriousLearner** [comment]
@RamolaWeb Yes, seems doable and a good practice. Once #15 is merged, this can be taken.

**Comment by CuriousLearner** [comment]
@RamolaWeb Now that #15 is merged in, you can start working on other issues, that are assigned to you. Let me know if you're stuck/ need any help.

As always, thank you for your contributions to the project.
